plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcWithBloomFilters.java

```diff
@@ -55,14 +56,50 @@ public void testBloomFilterPropertiesArePersistedDuringCreate()
                 "orc_bloom_filter_columns = array['a','b']," +
                 "orc_bloom_filter_fpp = 0.1)");
 
-        MaterializedResult actualProperties = computeActual("SELECT * FROM \"" + tableName + "$properties\"");
-        assertThat(actualProperties).isNotNull();
-        MaterializedResult expectedProperties = resultBuilder(getSession())
-                .row("write.orc.bloom.filter.columns", "a,b")
-                .row("write.orc.bloom.filter.fpp", "0.1").build();
-        assertContains(actualProperties, expectedProperties);
+        assertThat(getTableProperties(tableName))
+                .containsEntry("write.orc.bloom.filter.columns", "a,b")
+                .containsEntry("write.orc.bloom.filter.fpp", "0.1");
 
         assertThat((String) computeScalar("SHOW CREATE TABLE " + tableName))
                 .contains("orc_bloom_filter_columns", "orc_bloom_filter_fpp");
     }
+
+    @Test
+    void testBloomFilterPropertiesArePersistedDuringSetProperties()
+    {
+        String tableName = "test_metadata_write_properties_" + randomNameSuffix();
+        assertQuerySucceeds("CREATE TABLE " + tableName + "(A bigint, b bigint, c bigint)");
+
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES orc_bloom_filter_columns = ARRAY['a','B']");
+        assertThat(getTableProperties(tableName))
+                .containsEntry("write.orc.bloom.filter.columns", "a,b");
+
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES orc_bloom_filter_columns = ARRAY['a']");
+        assertThat(getTableProperties(tableName))
+                .containsEntry("write.orc.bloom.filter.columns", "a");
+
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES orc_bloom_filter_columns = ARRAY[]");
+        assertThat(getTableProperties(tableName))
+                .doesNotContainKey("write.orc.bloom.filter.columns");
+    }
+
+    @Test
+    void testInvalidBloomFilterProperties()
```

Comment:
Can we have a test for an Iceberg parquet table on which we're trying to set the `orc_bloom_filter_columns` property?